integration/kubernetes/operator/presto/config/rbac/role.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manager-role
+rules:
+- apiGroups:
+  - alluxio.com
+  resources:
+  - prestoclusters
```

Comment:
"apps" is required in apiGruops to avoid permission error. If "deployments", "pods", "configmaps" and "services" are not in the resource list, it may also cause permission error.
```yaml
- apiGroups:
  - apps
  - alluxio.com
  resources:
  - prestoclusters
  - configmaps
  - deployments
  - pods
  - services
```
![list_resoures_forbidden_snaphot](https://user-images.githubusercontent.com/6129818/169196507-8f6d5fd4-4204-4559-9a50-fd3c478af544.png)